app/controllers/application_controller.rb

```diff
@@ -12,4 +13,43 @@ def configure_permitted_parameters
    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :home_state])
   end
 
+  base_uri "https://en.wikipedia.org/w/api.php?format=json&action=query&prop=extracts&exintro&explaintext&redirects=1&titles="
```

Comment:
I think this is all in your parseWiki PORO now -- please remove from here!

Reply:
We removed the parseWiki PORO and implemented it's functionality inside of the application_controller.rb as the new state_info method.